Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -32,8 +32,7 @@ const SubmissionDetails = () => {
 
     const submissionsNavigationClassName = 'submissionsNavigation';
 
-    const submissionDetailsClass = 'submissionDetails';
-    const submissionDetailsClassName = concatClassNames(styles.navigation, submissionDetailsClass);
+    const submissionDetailsClassName = concatClassNames(styles.navigation, styles.submissionDetails);
```

Comment:
You sure about this?
From the convention, I guess `submissionDetails` is a class tha @MadjarovaML placed here for tests.
But, if you move it to `styles`, at build time it will be suffixed with build number, i.e. something like`submissionDetails-1b3f3`.
Better to leave both